moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -242,6 +242,16 @@ void MotionPlanningFrame::onFinishedExecution(bool success)
   // update query start state to current if neccessary
   if (ui_->start_state_combo_box->currentText() == "<current>")
     startStateTextChanged(ui_->start_state_combo_box->currentText());
+
+  // update previous state if plan is successfully executed
+  if (success)
+  {
+    planning_display_->updatePreviousState();
+    if (ui_->goal_state_combo_box->currentText() == "<previous>")
```

Comment:
Here, only the goal state is updated. Do we really need <previous> for the start state as well (see above)?